<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScreen</class>
 <widget class="QMainWindow" name="mainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yatdl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>549</width>
       <height>279</height>
      </rect>
     </property>
     <widget class="QListWidget" name="taskList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="newTaskButton">
       <property name="text">
        <string>New Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose_session"/>
   </widget>
   <widget class="QMenu" name="menuAdmin_tools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Admin tools</string>
    </property>
    <addaction name="actionRegistration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdmin_tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_session">
   <property name="text">
    <string>Close session</string>
   </property>
  </action>
  <action name="actionRegistration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
